setup.py

```diff
@@ -18,6 +18,23 @@
 
 from lizzy_client.version import VERSION
 
+requirements = [
+    'click==6.3',
```

Comment:
I think it should be ">=" instead of "==" unless we know of any issue with a more recent version.